cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -50,6 +50,11 @@
     final OnConsentUIFinishedCallback onConsentUIFinished;
     final OnConsentReadyCallback onConsentReady;
     final OnErrorCallback onError;
+    final pmWillShowCallback pmWillShow;
```

Comment:
should rename callbacks to android standards here as well

Reply:
Renamed callback methods as per android convention